src/VisualStudio/Xaml/Impl/Diagnostics/XamlDiagnosticIds.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Editor.Xaml.Diagnostics
+{
+    internal class XamlDiagnosticIds
+    {
+        public const string UnnecessaryNamespacesId = "XAML0001";
```

Comment:
These don't overlap with anything returned by the markup compiler do they?  Ideally they are unique enough to be good search targets to learn more about.